packages/terra-toolkit-docs/src/terra-dev-site/tool/stylelintConfigTerra/CustomRules/custom-property-name.tool.mdx

```diff
@@ -0,0 +1,3 @@
+import Readme from '@cerner/stylelint-config-terra/lib/rules/custom-property-name/README.md';
```

Comment:
Should this include the Badge?

Reply:
Should we put the badge on every page? 🤷‍♂️